src/backend/src/services/teams.services.ts

```diff
@@ -152,8 +159,20 @@ export default class TeamsService {
 
     if (team.dateArchived) throw new HttpException(400, 'Cannot edit the head of an archived team');
 
-    if (newHead && team.members.map((user) => user.userId).includes(newHead?.userId))
-      throw new HttpException(400, 'Error: Team head cannot be a member');
+    // if the new head is a current member on the team, this will remove them as a member
+    if (newHead && team.members.map((user) => user.userId).includes(newHead?.userId)) {
```

Comment:
Since you're checking that newHead exists, you should be able to drop the ? in the .includes (i.e. .includes(newHead.userId))

Reply:
> Since you're checking that newHead exists, you should be able to drop the ? in the .includes (i.e. .includes(newHead.userId))

Oh ok that makes sense, I'll fix that.